com.unity.shadergraph/Editor/Data/Attributes/ContextFilterableAttribute.cs

```diff
@@ -0,0 +1,9 @@
+using System;
+
+namespace UnityEditor.ShaderGraph
+{
+    [AttributeUsage(AttributeTargets.Class, AllowMultiple = true, Inherited = false)]
+    public abstract class ContextFilterableAttribute : Attribute
```

Comment:
Is this intended to be public?

Reply:
Originally I had thought it could be part of the node API, although I suppose internal would be the correct access mod. Will fix